### ISSUE TITLE
A11y_code.visualstudio.com(CELA-P1)_Get Started_Color Contrast: Color Contrast ratio for "Search docs" is 4.3:1

### ISSUE THREAD

**Issue by Arjun553** [microsoft/vscode-docs#2649]
**Check out Keros!** - Identify accessibility bugs before check-in and make bug fixing faster and easier.
**Environment Details:**
code.visualstudio.com/docs
Browser: Edge, Chrome

**Repro Steps:**
1)Hit the URL:https://code.visualstudio.com/docs
2)Open Color Contrast Analyzer
3)Verify color contrast ratio for contents , links, buttons and other controls of the page
4)Go to High Contrast Settings (Settings-> Colors-> High Contrast Settings) and check with all the four themes
5)Verify high contrast ratio for contents , links, buttons and other controls of the page

**Actual Result:**
Color Contrast ratio for "Search docs" is 4.3:1

**Expected Result:**
Color Contrast ratio for "Search docs" should be greater than or equal to 4.5:1

**User Impact:**
People with low vision users will face difficulty in understanding and accessing the controls if  Color Contrast ratio for "Search docs" is less than 4.5:1

**Mas reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

**Comment by Arjun553** [comment]
![Bug_2649_Color Contrast](https://user-images.githubusercontent.com/50437579/57450690-d06fa680-727c-11e9-8881-eaf15f2591a4.png)

**Comment by misolori** [comment]
Update `$placeholder-color` to `rgba(255, 255, 255, 0.8)` and that should pass on default and hover state:

![image](https://user-images.githubusercontent.com/35271042/59219554-a52bf000-8b77-11e9-953c-36a837bf51a1.png)

**Comment by gregvanl** [comment]
@Arjun553

**Comment by fsteffi** [comment]
GitHubTags:#CAITestPass_June’19;
